examples/sid_visualization/README.md

```diff
@@ -0,0 +1,135 @@
+
+# Ensure SID model is downloaded for deployment to Triton
+./scripts/fetch_data.py fetch models
+
+# Change to the example directory
+cd ${MORPHEUS_HOME}/examples/sid_visualization
+
+# Launch the containers
+DOCKER_BUILDKIT=1 docker-compose up --build -d
+```
+
+The following GUI should be displayed when all containers have completed launching:
+
+<img src="./img/initial_win.png" width=75% height=75%>
+
+### Build Morpheus
+
+Once docker-compose has been launched, exec into the container to build and run Morpheus:
+
+```bash
+# Exec into the morpheus container
+docker-compose exec morpheus bash
+
+# Inside the container, compile morpheus
+BUILD_DIR=build-docker ./scripts/compile.sh
+
+# Install morpheus with an extra dependency
+pip install -e . && pip install websockets
+
+# Verify Morpheus is installed
+morpheus --version
+
+# Ensure the data has been downloaded
+./scripts/fetch_data.py fetch examples
+```
+
+** Note: Keep this shell in the Morpheus Dev container running. It will be used later to start Morpheus.
+
+## Running the Demo
+
+### Running Morpheus
+
+After the GUI has been launched, Morpheus now needs to be started. In the same shell used to build Morpheus (the one running the Morpheus Dev container), run the following:
+
+```bash
+python examples/sid_visualization/run.py \
+  --debug --use_cpp=False --num_threads=1 \
+  --triton_server_url=triton:8001 \
+  --input_file=./examples/data/sid_visualization/group1-benign-2nodes-v2.jsonlines \
+  --input_file=./examples/data/sid_visualization/group2-benign-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group3-si-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group4-benign-49nodes.jsonlines
+```
+
+This launch will use all of the available datasets. Each dataset will show up as one batch in the visualization. Here is a description of each dataset:
+
+- `examples/data/sid_visualization/group1-benign-2nodes.jsonlines`
+  - Small scale with 2 nodes, no SID
+- `examples/data/sid_visualization/group2-benign-50nodes.jsonlines`
+  - Scale up to 50 nodes, no SID
+- `examples/data/sid_visualization/group3-si-50nodes.jsonlines`
+  - 50 nodes, with SID from a single node
+- `examples/data/sid_visualization/group4-benign-49nodes.jsonlines`
+  - Isolate bad node leaving 49 nodes, no SID
+
+The following is a screenshot after all four batches have been processed:
+
+<img src="./img/full_win.png" width=75% height=75%>
+
+Use the slider or the following buttons to step through the inferences batches in the visualization:
+|                              |                                                   |
+| ---------------------------- | ------------------------------------------------- |
+| <img src="./img/left.png">   | Step to previous inference batch                  |
+| <img src="./img/right.png">  | Step to next inference batch                      |
+| <img src="./img/replay.png"> | Step through all inference batches from beginning |
```

Comment:
Also, can you put a quick blub in here about:

- What the green spheres represent?
- What the moving dots represent?
- That you can hover over each packet to see its contents
- That "Simulating" is the Pause button to stop the animation